en/micro-integrator/docs/setup/deployment/deploying_wso2_ei.md

```diff
@@ -24,6 +24,18 @@ hostname = "localhost"
 
 Find more [parameters](../../../references/config-catalog/#deployment) for deployment settings.
 
+!!! info "Enable heartbeat messaging"
+    When a client is connected to the broker, both the broker and the client should be able to detect problem situations where the TCP connection is half open or where the connecting client/broker is unresponsive. This can be achieved by enabling heartbeat messaging between the broker and the client.
+    
+    The heartbeat messaging configuration allows both the broker and the client to verify whether the connection is inactive or whether the connecting system (broker or client) is inactive by periodically sending messages to each other.
+
+    | Parameter           | Description                                                                            | Default value |
+    |---------------------|----------------------------------------------------------------------------------------|---------------|
+    | `heartBeatInterval` | The time interval between heartbeat messages, specified in milliseconds.               | 5000          |
+    | `heartbeatMaxRetry` | The number of heartbeat messages sent to the client before terminating the connection. | 3             |
+
+    Note that this should only be used when your network latency **exceeds** `heartbeatMaxRetry * heartBeatInterval`.
```

Comment:
++
to set different heart beat level, we can use
-DheartBeatInterval = <value> and the same for environment variable.

Reply:
Please check the note I have added.